app/views/sign_up/registrations/new.html.erb

```diff
@@ -17,12 +17,30 @@
       html: { autocomplete: 'off', role: 'form' },
       url: sign_up_register_path) do |f| %>
     <%= f.input :email, label: t('forms.registration.labels.email'), required: true, input_html: { aria: { invalid: false } } %>
+
+    <fieldset class="usa-fieldset">
+      <legend class="sans-serif margin-bottom-1">
+        <%= t('forms.registration.labels.email_language') %>
+      </legend>
+
+      <p class="margin-bottom-4">
+        <%= t('account.email_language.languages_list',
+              app_name: APP_NAME,
+              list: I18n.available_locales.
+                map { |locale| t("account.email_language.name.#{locale}") }.
+                to_sentence(last_word_connector: " #{t('account.email_language.sentence_connector')} ")) %>
+      </p>
+
+      <%= render partial: 'shared/email_languages',
+                 locals: { f: f, selection: @register_user_email_form.email_language } %>
+    </fieldset>
+
     <span class='email-invalid-alert-inline usa-error-message margin-top-neg-3 margin-bottom-3 hide' role='alert' hidden='true'>
```

Comment:
In some unrelated work around these alerts, I just stumbled into a small problem from these changes:

Before|After
---|---
![before](https://user-images.githubusercontent.com/1779930/98402697-bfe8e980-2035-11eb-8bef-821025b29bb8.png)|![after](https://user-images.githubusercontent.com/1779930/98402706-c4150700-2035-11eb-8ceb-755894718acc.png)

The error text is supposed to be a hint accompanying the highlighted form field.

See also: https://design.login.gov/components/form-fields/#form-validation-errors

I don't think it's critical to fix, though I imagine it could be somewhat confusing for the error text to be shown so far away from the field.

Reply:
Great find, I'll fix